source/clear-linux/get-started/virtual-machine-install/vmw-player.rst

```diff
@@ -0,0 +1,313 @@
+
+#. Click the :guilabel:`Finish` button.
+
+Install Clear Linux into the new VM
+***********************************
+
+#. Select the newly-created VM and click the :guilabel:`Play virtual machine`
+   button. See Figure 9.
+
+   .. figure:: figures/vmw-player/vmw-player-09.png
+      :scale: 100%
+      :alt: VMware Workstation 14 Player - Power on virtual machine
+
+      Figure 9: VMware Workstation 14 Player - Power on virtual machine
+
+#. Follow the :ref:`install-on-target` guide to complete the installation of
+   |CL|.
+
+#. After the installation completes, reboot the VM. This reboot restarts the
+   |CL| installer.
+
+Detach the |CL| installer ISO from the VM
+*****************************************
+
+#. To enable the mouse pointer so you access VMware Workstation Player's
+   menus, press :kbd:`<CTRL>` + :kbd:`<ALT>` on the keyboard.
+
+#. To disconnect the CD/DVD to stop it from booting the |CL| installer ISO
+   again, click the :guilabel:`Player` menu. See Figure 10.
+
+   .. figure:: figures/vmw-player/vmw-player-10.png
+      :scale: 100%
+      :alt: VMware Workstation 14 Player - Edit CD/DVD settings
+
+      Figure 10: VMware Workstation 14 Player - Edit CD/DVD settings
+
+#. Go to :menuselection:`Removable Devices-->CD/DVD (IDE)-->Settings`.
+
+#. On the :guilabel:`Device status` section, uncheck the
+   :guilabel:`Connected` and the :guilabel:`Connect at power on` settings.
+   See Figure 11.
+
+   .. figure:: figures/vmw-player/vmw-player-11.png
+      :scale: 100%
+      :alt: VMware Workstation 14 Player - Disconnect CD/DVD
+
+      Figure 11: VMware Workstation 14 Player - Disconnect CD/DVD
+
+#. Click the :guilabel:`OK` button.
+
+#. To power off the VM, click the :guilabel:`Player` menu. See Figure 12.
+
+   .. figure:: figures/vmw-player/vmw-player-12.png
+      :scale: 100%
+      :alt: VMware Workstation 14 Player - Power off virtual machine
+
+      Figure 12: VMware Workstation 14 Player - Power off virtual machine
+
+#. Go to :guilabel:`Power` and select :guilabel:`Shut Down Guest`.
+
+Enable UEFI boot support
+************************
+
+|CL| needs UEFI support to boot. To enable UEFI, add the
+following line to the end of your VM's :file:`.vmx` file:
+
+.. code-block:: bash
+
+   firmware = "efi"
+
+.. note::
+
+   Depending on the OS, you can typically find the VMware VM files under:
+
+   * On Linux distros: :file:`/home/username/vmware`
+   * On Windows: :file:`C:\\Users\\username\\Documents\\Virtual Machines`
+
+Power on the VM
+***************
+
+After configuring the settings above, power on your |CL| virtual machine.
+
+#. On the :guilabel:`VMware Workstation Player` home screen, select your 
+   VM. See Figure 13.
+
+   .. figure:: figures/vmw-player/vmw-player-13.png
+      :scale: 100%
+      :alt: VMware Workstation 14 Player - Power on virtual machine
+
+      Figure 13: VMware Workstation 14 Player - Power on virtual machine
+
+#. Click :guilabel:`Play virtual machine`.
+
+Also see:
```

Comment:
For consistency and parallelism I would change this line to:
For other guides on using the VMWare player, see: